README.md

```diff
@@ -26,13 +27,13 @@ curl -s https://get.nextflow.io | bash
 
 Install Almanack:
 
-```
+```sh
```

Comment:
@aditigopalan this addition did not render. Recommend having someone review PRs before merging.